FirebaseDatabaseSwift/Tests/Codable/ServerValueCodingTests.swift

```diff
@@ -87,3 +87,114 @@ extension CurrencyAmount: ExpressibleByFloatLiteral {
     self.value = Decimal(value)
   }
 }
+
```

Comment:
I think I'm missing something here - none of these functions or classes are used anywhere above, are they?

Reply:
All except the bottom function are used by the `assertThat` implementation. These functions are uses more extensively in the Firestore tests and we may want to unify them.